packages/accounts-base/accounts_client.js

```diff
@@ -438,4 +453,24 @@ if (Package.blaze) {
   Package.blaze.Blaze.Template.registerHelper('loggingIn', function () {
     return Meteor.loggingIn();
   });
+
+  /**
+   * @global
+   * @name  loggingOut
+   * @isHelper true
+   * @summary Calls [Meteor.loggingOut()](#meteor_loggingout).
+   */
+  Package.blaze.Blaze.Template.registerHelper('loggingOut', function () {
```

Comment:
I think `Package.blaze` is good enough. This is what everyone is doing for weak dependencies.